control/pid_longitudinal_controller/src/pid_longitudinal_controller.cpp

```diff
@@ -456,6 +456,8 @@ PidLongitudinalController::ControlData PidLongitudinalController::getControlData
     current_interpolated_pose.first);
   control_data.nearest_idx = current_interpolated_pose.second + 1;
   control_data.target_idx = control_data.nearest_idx;
+  auto nearest_point = current_interpolated_pose.first;
```

Comment:
```suggestion
  const auto nearest_point = current_interpolated_pose.first;
```

Reply:
Yeah `nearest_point` is not updated later so it can be `const`.